sdv/multi_table/base.py

```diff
@@ -82,3 +83,74 @@ def update_table_parameters(self, table_name, table_parameters):
     def get_metadata(self):
         """Return the ``MultiTableMetadata`` for this synthesizer."""
         return self.metadata
+
+    def _validate_foreign_keys(self, data):
+        errors = []
+        for relation in self.metadata._relationships:
+            child_table = data[relation['child_table_name']]
+            parent_table = data[relation['parent_table_name']]
+            child_row = child_table[relation['child_foreign_key']]
+            parent_row = parent_table[relation['parent_primary_key']]
+            missing_values = child_row[~child_row.isin(parent_row)].unique()
+            if any(missing_values):
+                message = ', '.join(missing_values[:5].astype(str))
+                if len(missing_values) > 5:
+                    message = f'({message}, + more).'
+                else:
+                    message = f'({message}).'
+
+                errors.append(
+                    f"Error: foreign key column '{relation['child_foreign_key']}' contains "
+                    f'unknown references: {message}'
+                )
+
+        return errors
+
+    def validate(self, data):
+        """Validate data.
+
+        Args:
+            data (dict):
```

Comment:
I'd call it tables. Data makes me think it's just a dataframe

Reply:
I'm fine with tables, I just followed the issue specification, @amontanez24 green light on the `tables` rather than `data`?

Reply:
@npatki I think this was set to `data` to keep consistency with the `SingleTableSynthesizer` method, but since it does take a different type as input maybe it would make sense to change?

Reply:
I see the same issue with `tables` though -- it does not imply anything about this being a dictionary. In fact, I'd assume it's a **list** if I just saw the word.

To me, `data` does not imply a `DataFrame` but rather some concept of the collected data. The nice thing about using `data` is that whenever we do a demo to future customers, etc. it's very clear what the concept is.